sagemaker-python-sdk/mxnet_gluon_mnist/mnist.py

```diff
@@ -41,8 +41,10 @@ def train(channel_input_dirs, hyperparameters, **kwargs):
     # Collect all parameters from net and its children, then initialize them.
     net.initialize(mx.init.Xavier(magnitude=2.24), ctx=ctx)
     # Trainer is for updating parameters with gradient.
+    store = kv.create('dist_sync' if len(hosts) > 1 else 'local')
```

Comment:
thanks for this PR!

instead of importing kv, it would be simpler to just use the `str` version:

`kvstore = 'dist_sync' if len(hosts) > 1 else 'local'`

also, if you use `dist_device_sync` and `device` instead, then this will use device mode when gpus are available, and fall back to cpu (dist_sync or local) when they aren't.

Reply:
Done. Also fixed in `mxnet_gluon_sentiment/sentiment.py`